plugins/modules/vmware_vm_info.py

```diff
@@ -326,6 +329,17 @@ def get_virtual_machines(self):
             if self.module.params.get('show_tag'):
                 vm_tags = self.get_tag_info(vm)
 
+            storage_allocated = 0
+            for device in vm.config.hardware.device:
+                if isinstance(device, vim.vm.device.VirtualDisk):
+                    storage_allocated += device.capacityInBytes
+
+            allocated = {
```

Comment:
Add a flag to return these facts optionally (for example - `show_allocated` ) rather than always returning facts.

Reply:
Yes I will do this.